hydra-chain-observer/src/Hydra/Ouroborus/ChainObserver.hs

```diff
@@ -1,3 +1,4 @@
+{-# OPTIONS_GHC -Wno-deprecations #-}
```

Comment:
I am stuck trying to fix these deprecation errors. We could merge with this to unblock other things.